src/wwwroot/services/utils.js

```diff
@@ -148,5 +150,15 @@
       $window.localStorage.setItem('lang', key);
     }
 
+    function hasDifferentDomain() {      
+      return !(/relay/.test($window.location.host));
```

Comment:
If the domain is girerelay.com it will not work fine, right?

Reply:
Yes, it will not work fine :(